include/bs/kazwingwhack.h

```diff
@@ -40,4 +40,4 @@ extern f32 func_800F1DF4(f32[3], f32[3]);
 extern f32 func_800F3780(f32, s32[], s32);
 extern void func_800F3BD0(PlayerState *self, s32, f32[3]);
 
-#endif // __BS_KAZTORPEDO_H__
```

Comment:
The header guard for this file wasn't added in this PR, but while reviewing this I noticed that it has the wrong macro (`__BS_KAZTORPEDO_H__` instead of `__BS_KAZWINGWHACK_H__`).